### PR TITLE
Add tag for content

### PR DESCRIPTION
**Pull request by cnevinc**

### CODE REVIEW

File: src/main/resources/templates/publish.html [hunk 1]
```diff
@@ -25,18 +25,19 @@ <h1>Content Upload</h1>
             <td>locale</td>
             <td>
                 <select name="locale">
-                    <option value="en-IN">[en-IN] Shopping/Travel</option>
-                    <option value="id-ID">[id-ID] Shopping/Travel</option>
-                    <option value="zh-HANT">[zh-HANT] Travel</option>
-                    <option value="all">[ALL] Games/Travel</option>
+                    <option value="all">[all] ONLY for games</option>
+                    <option value="en-IN">[en-IN] Specify to en-IN [shopping/travel]</option>
```
Comment (cnevinc) [inline src/main/resources/templates/publish.html]: we may use it only for shopping.
and use "eng" for travel

File: src/main/java/org/mozilla/msrp/platform/vertical/content/ContentController.kt [hunk 2]
```diff
@@ -45,11 +45,11 @@ class ContentController @Inject constructor(private val contentService: ContentS
             })
 
 
-    @RequestMapping("/api/v1/content")
+    @GetMapping("/api/v1/content")
     fun getContent(
             @RequestParam(value = "category") category: String,
             @RequestParam(value = "locale") locale: String,
-            @RequestParam(required = false) tag: String = ""
+            @RequestParam(required = false) tag: String? = ""
```
Comment (benjamin-cheng) [inline src/main/java/org/mozilla/msrp/platform/vertical/content/ContentController.kt]: Could use `defaultValue = ""` to get rid of null-able tag string
https://github.com/mozilla-tw/mango_backend/blob/develop/src/main/java/org/mozilla/msrp/platform/vertical/video/VideoController.kt#L31

**Comment by cnevinc** [comment]
@Daisy-pliu We also need to add a new DB index on Nightly and production

File: src/main/java/org/mozilla/msrp/platform/vertical/content/ContentController.kt (hunk 2 above)
Comment (cnevinc) [reply src/main/java/org/mozilla/msrp/platform/vertical/content/ContentController.kt]: hmm..I want it to be nullable. So the default value can be removed.

**Pull request merged**